zebra/zebra_vxlan.c

```diff
@@ -6277,12 +6276,9 @@ void vxlan_vni_state_change(struct zebra_if *zif, uint16_t id, uint8_t state)
 		if (IS_ZEBRA_DEBUG_VXLAN)
 			zlog_debug("Cannot find VNI for VID (%u) IF %s for vlan state update",
 				   id, zif->ifp->name);
-		frrtrace(3, frr_zebra, netlink_intf_err, zif->ifp->name, id, 3);
+
 		return;
 	}
-	if (zif)
```

Comment:
Are we removing tracing stuff on purpose?

Reply:
Ahh, it was a copy-paste error from out internal repo. Fixed it